addon/components/variable-number/number.ts

```diff
@@ -44,7 +44,18 @@ export default class VariableNumberPluginNumberComponent extends Component<Args>
     return this.args.node;
   }
   get formattedNumber() {
-    return this.node.attrs.value as string;
+    const writtenNumber = Boolean(this.writtenNumber);
+
+    if (
+      !writtenNumber ||
+      Number.isNaN(Number(this.node.attrs.value)) ||
+      this.node.attrs.value === null ||
+      this.node.attrs.value === ''
+    ) {
+      return this.node.attrs.value as string;
+    } else {
+      return n2words(Number(this.node.attrs.value), { lang: 'nl' });
```

Comment:
I personally like the number more with "includeOptionalAnd" set to true, but I asked in jira if Bart knows of any preference, because both are officially correct "Belgian Dutch".

An actual config for this could later be added together with the idea of a "document language" (which Arne has talked about). But for now we need to at least decide on a default.


There is also the option "noHundredPairs" (note: the wiki docs are wrong, this is the correct spelling of the option :) ), but I can't find a source online what the correct spelling would be... I also asked this to Bart.

Reply:
Bart has answered (and probably the final answer):
- No "en" in words
- don't care about the hundredPairs

So both config options can stay their default (which is false)